### DISCUSSION TITLE
supabase db remote commit - doesn't capture indexes

### DISCUSSION THREAD

**Discussion by tholder** [supabase/supabase#13838] [Questions]
I'm just setting up local dev, I have some triggers on my remote DB.

```
supabase db remote commit
```

This has pulled down the schema but it seems to miss pulling down triggers. Is this as expected?

**Comment by GaryAustin1** [comment]
Your title says indexes but then you refer to triggers....

Are these public schema triggers or auth triggers?
https://github.com/supabase/cli/issues/120
Supabase schema triggers don't get copied.
